pdp/attribute.go

```diff
@@ -89,13 +89,13 @@ func (v AttributeValueType) describe() string {
 	case DataTypeSetOfNetworks:
 		items := []string{}
 		i := 0
-		for _, n := range v.Value.([]net.IPNet) {
+		for n := range v.Value.(*SetOfNetworks).Iterate() {
```

Comment:
Any AttributeValueType instance should be created in the way that its DataType and Value fields are consistent otherwise it means issue with some of parsers we provide as part of pdp or pdpserver. The only option we have in the case - panic.